packages/frontend/src/components/UserSettings/UsersAndGroupsPanel/UsersView.tsx

```diff
@@ -275,10 +324,13 @@ const UsersView: FC = () => {
     const { user } = useApp();
     const { classes } = useTableStyles();
 
+    // TODO: this is a feature flag while we are building groups.
+    const groupManagementEnabled = useFeatureFlagEnabled('group-management');
+
     const [search, setSearch] = useState('');
 
     const { data: organizationUsers, isLoading: isLoadingUsers } =
-        useOrganizationUsers(search);
+        useOrganizationUsers({ searchInput: search, includeGroups: 1000 });
```

Comment:
is 1000 intentional?

Reply:
Yea, but I'll put a TODO. And maybe make it 10K. ATM these endpoints require a number, but we aren't doing pagination